src/core_modules/capture-core/components/Pages/Search/SearchDomainSelector/SearchDomainSelector.js

```diff
@@ -0,0 +1,88 @@
+// @flow
```

Comment:
this is the component we wanted to extract from a previous PR. 
find the comment of this conversation here https://github.com/dhis2/capture-app/pull/822#discussion_r450699715

Reply:
Yeah havent realised we do that. Will rename it 👍